grassroots-backend/package.json

```diff
@@ -7,9 +7,7 @@
   "license": "UNLICENSED",
   "scripts": {
     "build": "nest build",
-    "start": "nest start",
-    "start:dev": "nest start --watch",
-    "start:debug": "nest start --debug --watch",
+    "start": "nest start --watch --debug",
```

Comment:
We talked about having `npm start` be the "perfect" dev command. I know that's not actually possible but this is my attempt.

Reply:
Actually... we didn't. It was on the agenda, but not discussed.

Reply:
LGTM

I wonder if the `test` commands should default to watching as well?

That would require splitting the tstyche tests into a separate command, but might still be worth it.

Reply:
done